core/src/main/java/io/questdb/griffin/engine/groupby/GroupByUtils.java

```diff
@@ -176,6 +176,18 @@ public static void prepareGroupByRecordFunctions(
                         case ColumnType.LONG256:
                             fun = Long256Column.newInstance(keyColumnIndex - 1);
                             break;
+                        case ColumnType.GEOBYTE:
+                            fun = new GeoByteColumn(keyColumnIndex - 1, type);
```

Comment:
could we implement factory method on `Geo*Column` classes, similar to what we have here:

![image](https://user-images.githubusercontent.com/7276403/152987242-778cb08c-ecb2-499b-9d24-dcd28044c2f7.png)